openfl/component/aggregator/aggregator.py

```diff
@@ -510,7 +512,7 @@ def _process_named_tensor(self, named_tensor, collaborator_name):
             tuple(named_tensor.tags)
         )
         tensor_name, origin, round_number, report, tags = tensor_key
-        assert ('compressed' in tags or 'lossy_decompressed' in tags), (
+        assert ('compressed' in tags or 'lossy_compressed' in tags), (
```

Comment:
We use enumerated lists like OptTreatment.RESET. Do you think we should do the same here? It's your choice. I was thinking of CompressionType.Lossy or something like that.

Reply:
Fair question. The primary argument that I have against a dedicated enum for compression is that it complicates the tensorkey definition. The reason this label is maintained in tags instead of a dedicated field is that compression is typically only applied during tensor transport. To be consistent with the other tags ('delta', 'model', etc.) it should probably be kept as is for now. I'm not opposed to reformulating this later though

Reply:
No worries.